run--01d--prep-acs5-data.Rmd

```diff
@@ -199,9 +202,154 @@ acs5tract_test <-
 #View(acs5tract_test %>% select(wgt, n, r, incpov_r0to50_est, incpov_r0to50_est))
 ```
 
-
 ```{r}
 save(acs5tract, acs5puma,
      file = glue("{output_path}acs5_data_{my_output_tag}.Rda"))
 ```
 
+```{r prepare county argument for fresh pull of acs5 data}
+if (exists("my_county_fip")) county_val <- my_county_fip else county_val <- NULL
+```
+
+```{r fresh pull of both income and work status tables}
+# "Age by Ratio of Income to Poverty Level In the Past 12 Months"
+acs5_incpov_byage_raw    <- pull_table("B17024", state = my_state_abbr, county = county_val, year = acs5_year)
+
+# "Relationship To Householder For Children Under 18 Years In Households"
```

Comment:
Can we change the definition of B23007 in line 218 to "Presence of Own Children Under 18 Years by Family Type by Employment Status". Current description is for B09018